src/current/_includes/releases/cloud/2024-02-05.md

```diff
@@ -0,0 +1,5 @@
+## February 5, 2024
+
+<h3 id="2024-02-05-console-changes"> Console changes </h3>
+
+- CockroachDB {{ site.data.products.dedicated }} clusters now have a [**Metrics** page]({% link cockroachcloud/metrics-page.md %}) in the Console with charts to **Monitor SQL Activity** and **Identify SQL Problems**. On the **Metrics** page, a **Custom** tab takes you to the [**Custom Metrics Chart** page]({% link cockroachcloud/custom-metrics-chart-page.md %}) (available in [preview]({% link {{site.current_cloud_version}}/cockroachdb-feature-availability.md %})) where you can create custom charts showing the time series data for an available metric or combination of metrics.
```

Comment:
```suggestion
- A new [custom **Custom Metrics Chart** page]({% link cockroachcloud/metrics-page.md %}) has been added in the CockroachDB Cloud Console for CockroachDB {{ site.data.products.dedicated }} clusters, with charts to both monitor SQL activity and identify SQL problems. From the **Metrics** page, you can click the **Custom** tab to go to the [**Custom Metrics Chart** page]({% link cockroachcloud/custom-metrics-chart-page.md %}), where you can create custom charts showing time-series data for each available metric or combination of metrics. This new functionality is available in [preview]({% link {{site.current_cloud_version}}/cockroachdb-feature-availability.md %})).
```
WDYT about this edit? This keeps the page name in sync with https://github.com/cockroachdb/docs/pull/18268/files#diff-b1d94ad8f8a151ec24d25b5be2a226c01006f4176f3d505b72214d4a9ddf5156R7.

Reply:
I think I need to keep my original sentence because Metrics is a new page for dedicated:
<img width="720" alt="Screenshot 2024-02-05 at 2 00 33 PM" src="https://github.com/cockroachdb/docs/assets/58752716/b6cf32f5-cfaa-4f49-91e5-e1cce756f5a6">